lib/serve.js

```diff
@@ -221,7 +221,7 @@ function prebuild(options) {
 
 function createModelRouter(modelConfig, options, config, addBeforeReadyTask, addScheduleTask, routes = {}) {
     const { slug } = modelConfig;
-    const cacheFilename = getCacheFilename(modelConfig);
+    const cacheFilename = getCacheFilename(modelConfig, true);
```

Comment:
Something wrong in this logic now. Suppose we need to relay on `modelConfig.cache` settings instead of generation for a filename here.
And latter `path.relative(process.cwd(), cacheFilename)` should not provide exact filename, since it changes, but pattern like `model-name.[timestamp].json` (or how it looks like)